tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -579,6 +584,21 @@ where
 					Ok(TrustedCallResult::Streamed)
 				}
 			},
+
+			TrustedCall::remove_identity(signer, who, identity) => {
+				debug!("remove_identity, who: {}", account_id_to_string(&who));
+
+				Self::remove_identity_internal(
```

Comment:
Maybe we don't need the extra `*internal` function as it's a bit too verbose - just to increase the readability :)